cmd/mt-index-cat/main.go

```diff
@@ -237,7 +238,7 @@ func main() {
 		}
 	}
 
-	var defs []schema.MetricDefinition
+	var defs []indx.MetricDefinition
```

Comment:
This might be a thing of personal taste... but i think i would either import that as the full word `index`, or if you want to shorten it then name it `idx` to be consistent with the package name. but `indx` is a weird hybrid of shortened / full

Reply:
+1 lets always import that package with its canonical name, idx so we don't need override the name. this helps with code searches
that does mean we can't use the variable name idx to hold the actual index instance. so let's use `metricIndex` which is the same as what metrictank does.

Reply:
@robert-milan : 

> +1 lets always import that package with its canonical name, idx 

so instead of:
```
metricIndex "github.com/grafana/metrictank/idx"
```
do
```
"github.com/grafana/metrictank/idx"
```